app/controllers/geo/GeoElasticsearch.java

```diff
@@ -12,67 +12,89 @@
 import org.elasticsearch.action.admin.indices.create.CreateIndexRequestBuilder;
 import org.elasticsearch.action.admin.indices.refresh.RefreshRequest;
 import org.elasticsearch.client.Client;
-import org.elasticsearch.node.Node;
 import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.node.Node;
 
 import com.typesafe.config.Config;
 import com.typesafe.config.ConfigFactory;
 
+/**
+ * Configuration and creation of Elasticsearch index
+ */
 @SuppressWarnings("javadoc")
```

Comment:
Ignoring missing Javadocs is fine for test classes, but for production code the reason for the custom compiler check is to actually ensure all public API is documented, so we should add documentation here.